packages/protocol/test/stability/reserve.ts

```diff
@@ -183,6 +183,12 @@ contract('Reserve', (accounts: string[]) => {
   describe('#getOrComputeTobinTax()', () => {
     let mockStableToken: MockStableTokenInstance
 
+    const expectedNoTobinTax: [BN, BN] = [new BN(0), new BN(10).pow(new BN(24))]
```

Comment:
Can we add a test for getting the reserve balance when other reserve addresses have been added?